common/tests/test_util.cc

```diff
@@ -38,7 +42,7 @@ TEST_CASE("util::read_file") {
     std::string content = random_bytes(64 * 1024);
     write(fd, content.c_str(), content.size());
     std::string ret = util::read_file(filename);
-    REQUIRE(ret == content);
+    REQUIRE(hasher(ret) == hasher(content));
```

Comment:
asserting the whole files gets dumped into the XML file, easier to just compare the hashes

Reply:
done in another PR